app/lib/bing_api_v7.rb

```diff
@@ -0,0 +1,43 @@
+class BingApiV7
+  BING_URI = 'https://api.cognitive.microsoft.com'.freeze
+  BING_PATH = '/bing/v7.0/search'.freeze
+
+  def initialize(element)
+    @element = element
+  end
+
+  def uri(*query)
+    URI(BING_URI + BING_PATH + "?q=" + CGI.escape(query[0] || ''))
+  end
+
+  def search(options = {})
+    request = Net::HTTP::Get.new(uri(options[:query]))
+    request['Ocp-Apim-Subscription-Key'] = api_key
+
+    response = Net::HTTP.start(uri.host, uri.port, use_ssl: uri.scheme == 'https') do |http|
```

Comment:
this isn't gonna work..

Reply:
but it is)

Reply:
i meant that you should fetch uri components from current uri that you are using for company, instead of generating new